PEM 8.4 release branch (#2466)
EnterpriseDB/docs: product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/12_pem_manage_probes/03_pem_probe_config/01_pem_probes.mdx

A `probe` is a scheduled task that retrieves information about the database objects that are being monitored by the PEM agent. PEM uses the collected information to build the graphs displayed on each homepage. The `Manage Probes` tab (accessed via the `Management` menu) allows you to modify the data collection schedule and the length of time that PEM will retain information returned by a specific probe.

> Unless otherwise noted, Postgres Enterprise Manager enables the probes listed in the table below at server, database, schema, extension(from PEM 8.4 onwards), or agent levels:

Reviewer: ```suggestion
Unless otherwise noted, Postgres Enterprise Manager enables the following probes at the server, database, schema, extension (starting with version 8.4), or agent levels:
```

Reviewer: Are there other levels? If this is a comprehensive list we might consider having a New Feature note about extensions. Might make sense to do that regardless. For the table, I would remove "This probe" from the start of each description and start with the verb instead.

| Background Writer Statistics             | This probe monitors information about the background writer. The information includes:<br />-   The number of timed checkpoints     <br /> -   The number of requested checkpoints     <br /> -   The number of buffers written (by checkpoint)     <br /> -   The number of buffers written (by background writer)     <br /> -   The number of background writer cycles     <br /> -   The number of background buffers written     <br /> -   The number of buffers allocated     <br />                                                                                                                                                                                                                                    | Server                              |

> | Barman Configuration                     | This probe returns information about Barman tool global configuration.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Agent                    |

Reviewer: add "the" after "about" in these descriptions

Author: done in a separate commit

## BDR Probes

To monitor the BDR Group via [BDR dashboards](../../01_dashboards/03_bdr_dashboard/#bdr_dashboard) the following probes must be enabled. All these probes are configured at extension level.

! Note

>   Prior to version 8.4, all these probes are available at server level.

Reviewer: does this mean that these probes are only available at the extension level now or both server and extension?

Author: Prior to version 8.4, all these probes are available at the server level only. From 8.4 onwards they are available at the only extension level.
